templates/ListFacet.mustache

```diff
@@ -0,0 +1,41 @@
+{{#hasToggle}}
+	<div class="wikibase-faceted-search__facet-toggle">
+		<a class="wikibase-faceted-search__cdx-button-link" href="">
+			<button class="cdx-button cdx-button--action-progressive cdx-button--weight-primary">
+				{{msg-and}}
+			</button>
+		</a>
+		<a class="wikibase-faceted-search__cdx-button-link" href="">
+			<button class="cdx-button">
+				{{msg-or}}
+			</button>
+		</a>
+	</div>
+{{/hasToggle}}
+
+{{#checkboxes}}
+	<div
+		class="wikibase-faceted-search__facet-item wikibase-faceted-search__facet-item-checkbox cdx-checkbox">
+		<a class="wikibase-faceted-search__facet-item-link" href="{{link}}">
+
+			<div class="cdx-checkbox__wrapper">
+				<input
+					id="{{id}}"
+					class="cdx-checkbox__input"
+					type="checkbox"
+					{{#checked}}checked{{/checked}}
+					{{! Prevent input from being interacted by keyboard}}
+					tabindex="-1"
+				/>
+				<span class="cdx-checkbox__icon"></span>
+				<div class="cdx-checkbox__label cdx-label">
+					<label class="cdx-label__label" for="{{id}}">
+						{{! Change to <div> tag instead of <span> tag for valid HTML }}
+						<span class="cdx-label__label__text">{{>FacetItemLabel}}</span>
```

Comment:
> For some reason the counts no longer show up,

In `FacetItemLabel.mustache` we can remove the `{{#count}}` condition.

![Screenshot_20250113_130941](https://github.com/user-attachments/assets/6136392d-4626-44fa-9c9e-73da34c8a553)


I don't think we need that any more, since any list facet value that we send to the template must have a count (`ValueCount $count` is not null). If it has 0 count then we shouldn't display the facet value (and even if we had to, the count is still a number).